validation_run.py

```diff
@@ -0,0 +1,182 @@
+# This code is used to generate simulations using parameters from the single outbreaks. Each set of parameters will generate 5000 valid results (We only consider the simulations with number of introductions on day 1 > 0)
+# Please note that the code below requires significant amount of computational resources since in order to generate valid results, the number of total simulations can reach to 10E13.
+
+## import libraries
+import numpy as np
+import pandas as pd
```

Comment:
Doesn't seem like these libraries are required. Suggest cleaning up imports. I believe only `numpy` and `covid_workplace_model` may be sufficient.